src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/AuthModal.java

```diff
@@ -42,14 +46,21 @@ private void switchToMainWindowHandle() {
 
   public boolean isOpened() {
     switchToAuthModalHandle();
-    boolean isOpenedResult = authModal.isDisplayed();
+    boolean isOpenedResult = registerAuthModal.isDisplayed();
```

Comment:
I see potential risk that it will be false, because it wasn't displayed YET... I'd suggest adding try for waiting for element being visible, switching to main window handle and then returning true (or registerAuthModal.isDisplayed() ) or catching NoSuchElementException - switching to main window and returning false.

What do you think?